contrib/exporters/secadvisor/mod/resolve_multi.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"github.com/skydive-project/skydive/common"
+)
+
+type resolveMulti struct {
+	resolvers []Resolver
+}
+
+// NewResolveMulti creates a new name resolver
+func NewResolveMulti(resolvers ...Resolver) (interface{}, error) {
```

Comment:
we are not returning any error at any point, still do we need the ``` error ``` in return type?

Reply:
fixed